### PR TITLE
Link fixed..

### PR DESCRIPTION
**Pull request by kalyankrishna1**
updated sample.json incase we need to do a major update later

### CODE REVIEW

File: AppCreationScripts/Cleanup.ps1 [hunk 1]
```diff
@@ -49,22 +50,23 @@ This function removes the Azure AD applications for the sample. These applicatio
     Write-Host "Cleaning-up applications from tenant '$tenantName'"
 
     Write-Host "Removing 'service' (TodoListService) if needed"
-    $app=Get-AzureADApplication -Filter "identifierUris/any(uri:uri eq 'https://$tenantName/TodoListService')"  
+    $app=Get-AzureADApplication -Filter "DisplayName eq 'TodoListService'"
```
Comment (jmprieur) [inline AppCreationScripts/Cleanup.ps1]: I disagree on this, @kalyankrishna1 
there are so many apps named TodoListService that you'd risk to remove the wrong one (that has happened this week actually :( ). But it's true that now that the App ID URI is api://'+$serviceAadApplication.AppId, this is wrong

We probably have a name unique to the sample.

File: AppCreationScripts/Steps.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
+
```
Comment (jmprieur) [inline AppCreationScripts/Steps.md]: You should not commit the Steps.md

File: AppCreationScripts/sample.json [hunk 3]
```diff
@@ -12,13 +13,16 @@
...
+      "Kind": "WebApi",
       "IsPublicClient": false,
-      "HomePage": "https://localhost:44321/"
+      "HomePage": "https://localhost:44321/",
+      "ReplyUrls": "https://localhost:44321/"
     },
     {
       "Id": "client",
-      "Name": "TodoListDaemon",
+      "Name": "TodoList_daemon_v1",
```
Comment (jmprieur) [inline AppCreationScripts/sample.json]: we might want to name it todoList_web_daemon_v1?

File: AppCreationScripts/sample.json [hunk 4]
```diff
@@ -12,13 +13,16 @@
   "AADApps": [
     {
       "Id": "service",
-      "Name": "TodoListService",
+      "Name": "TodoListService_daemon_v1",
```
Comment (jmprieur) [inline AppCreationScripts/sample.json]: we might want to name it todoListService_web_daemon_v1;
the reason is people are quite polarized on this.
there are some customers who want the simple console daemon, and other who want the complex web daemon sample :)

File: AppCreationScripts/sample.json [hunk 5]
```diff
@@ -48,7 +52,7 @@
         },
         {
           "key": "ida:Audience",
-          "value": "service.IdentifierUris"
+          "value": "$serviceIdentifierUri"
```
Comment (jmprieur) [inline AppCreationScripts/sample.json]: why did you change this, @kalyankrishna1 
wasn't it working ?

**Comment by jmprieur** [comment]
Shall I let you merge, @kalyankrishna1 ?

File: AppCreationScripts/sample.json (hunk 3 above)
Comment (kalyankrishna1) [reply AppCreationScripts/sample.json]: Done

File: AppCreationScripts/sample.json (hunk 4 above)
Comment (kalyankrishna1) [reply AppCreationScripts/sample.json]: Done

File: AppCreationScripts/sample.json (hunk 5 above)
Comment (kalyankrishna1) [reply AppCreationScripts/sample.json]: You were correct, fixed
